CMakeLists.txt

```diff
@@ -59,11 +59,12 @@ if(NOT COMPILER_SUPPORTS_C17)
   message(FATAL_ERROR "The compiler ${CMAKE_C_COMPILER} has no C17 support. Please use a different C compiler.")
 endif()
 
-find_package(Libev 4.11)
-if (LIBEV_FOUND)
-  message(STATUS "libev found")
-else ()
-  message(FATAL_ERROR "libev needed")
+
+option(USE_URING "Use either io_uring or epoll as Linux event handling backend" ON)
```

Comment:
We should detect liburing, and if it is installed - plus supported - then both io_uring and epoll will be available

Reply:
So should it be an option in the config file to enable one or the other?

Reply:
Yes, see the current `libev` option - so yours would be `auto`, `io_uring`, `epoll` and `kqueue`.

If the liburing version isn't support then `io_uring` would do a fall-back to `epoll`

Reply:
You can rename `libev` to `network` or a better name if you have one

Reply:
`libev backend` is used extensively in the documentation. I could use simply `ev backend` and `ev` or `pgagroal_ev` to replace the `libev` option. I think `network` is fine too. Your call.

Reply:
You could use "network backend" in the documentation, and then just `network` for the configuration option

Reply:
Good!

Reply:
I prefer `ev_backend` other `network`, but I guess the choice has been made.

Reply:
Either way - as long as it isn't `libev` or `libevent` related

Reply:
I guess `ev_backend` then? I believe it to be more consistent with the existing code either way...

Reply:
Yes, `ev_backend` is ok